third_party/terraform/tests/resource_pubsub_topic_test.go

```diff
@@ -40,6 +41,37 @@ func TestAccPubsubTopic_update(t *testing.T) {
 	})
 }
 
+func TestAccPubsubTopic_cmek(t *testing.T) {
```

Comment:
For this handwritten test, we can use https://github.com/GoogleCloudPlatform/magic-modules/blob/caf402439790e7bb06d94a0d2fa4ff76c035191e/third_party/terraform/utils/bootstrap_utils_test.go#L21-L25 to provide a KMS key.

Reply:
Cool, will look into it.

Reply:
Indeed much nicer, changes pushed.